src/components/VehiclesColumnProvider.js

```diff
@@ -33,11 +34,6 @@ export default function VehiclesColumnProvider({ children, currentBook }) {
       { label: "Silhouette", name: "silhouette" },
       { label: "Speed", name: "speed" },
       { label: "Handling", name: "handling" },
-      { label: "Armor", name: "armor" },
```

Comment:
Looks like these columns were unintentionally removed.

Reply:
Oohhh yeah, didn't notice that. Git failed miserably at trying to merge a few of these and basically just highlighted half the column object as the "change".